posthog/caching/test/test_insight_cache.py

```diff
@@ -112,7 +112,7 @@ def test_update_cache(cache, team: Team, user: User):
 
 @pytest.mark.django_db
 @freeze_time("2020-01-04T13:01:01Z")
-def test_update_cache_updates_identical_cache_keys(cache, team: Team, user: User):
+def test_update_cache_updates_identical_cache_keys(team: Team, user: User, cache):
```

Comment:
this is a 🤯 fix:

We were getting an extra team in the cache here because the cache was cleared before the team fixture was called, which was adding the team to the cache result.

cc: @pauldambra for when you work on insight caching again and _might_ bang your head against this. Looks soo harmless, order of args 😂